docs/quickstart_guide.md

```diff
@@ -67,21 +43,41 @@ We tested the following instructions by setting up a **2-node** cluster on Cloud
 
 SSD-equipped nodes are highly recommended. Full list of CloudLab nodes can be found [here][cloudlab-hw].
 
+### 4. Go Installation
+
+If you intend to build the setup scripts from source, you can refer to the [Go Installation Guide](https://go.dev/doc/install) for installing Go on your system.
+
+- Confirm the installation:
+   ```bash
+   go version
+   ```
+
+**We highly recommend you install Go with version `1.19`**
```

Comment:
Won't it make go being installed twice: here and during setup_node script? And that actually makes little sense to install go from go tool.

Reply:
I agree with that. Leonid, could you fix up and go ahead with merging this PR? sync up with Ruiqi about the runners

Reply:
@leokondrashov